docs/pages/identity-governance/aws-iam-identity-center/guide.mdx

```diff
@@ -113,6 +115,21 @@ the same [IdP](../../zero-trust-access/sso/sso.mdx) as your current Identity Cen
 external identity source.
 </Admonition>
 
+<Admonition type="info" title="SAML IdP RBAC">
+In this step, Teleport creates a `saml_idp_service_provider` resource for the 
+AWS IAM Identity Center. 
+
+Starting with Teleport version 18 and role version 8, the SAML IdP service provider 
+resources are gated behind `app_labels` label matchers. If you are using Teleport 
```

Comment:
I think it's pretty safe to say that anyone following this guide anew will be on Teleport 18 and role version v8. Should we move this out of the admonition and just make a part of the base guide?